tools/ops/mugc.py

```diff
@@ -155,6 +168,9 @@ def setup_parser():
     parser.add_argument(
         "--prefix", default="custodian-",
         help="The Lambda name prefix to use for clean-up")
+    parser.add_argument(
+        "--policy-regex", default="custodian-.*",
```

Comment:
this probably needs a leading ^ so it only matches from the beginning of the string.

```python
>>> import re
>>> re.search("custodian-.*", 'not-custodian-x')
<re.Match object; span=(4, 15), match='custodian-x'>
```

Reply:
Yep you're right about that